storage/pebble/value_cache.go

```diff
@@ -0,0 +1,148 @@
+package pebble
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/cockroachdb/pebble"
+	lru "github.com/hashicorp/golang-lru/v2"
+
+	"github.com/onflow/flow-go/module"
+	"github.com/onflow/flow-go/storage"
+)
+
+func withLimit[K comparable, V any](limit uint) func(*Cache[K, V]) {
+	return func(c *Cache[K, V]) {
+		c.limit = limit
+	}
+}
+
+type storeFunc[K comparable, V any] func(key K, val V) func(pebble.Writer) error
+
+//	func withStore[K comparable, V any](store storeFunc[K, V]) func(*Cache[K, V]) {
```

Comment:
any reason to leave this commented out?

Reply:
because it's not being used by any code yet, and will be needed when moving other badger modules to pebble.